MapboxSearch/offline/src/main/java/com/mapbox/search/offline/OfflineSearchCallbackAdapter.kt

```diff
@@ -7,8 +7,10 @@ import com.mapbox.search.base.result.BaseSearchResult
 internal class OfflineSearchCallbackAdapter(private val callback: OfflineSearchCallback) : BaseSearchCallback {
     override fun onResults(results: List<BaseSearchResult>, responseInfo: BaseResponseInfo) {
         callback.onResults(
-            results.map { OfflineSearchResult(it.rawSearchResult) },
-            OfflineResponseInfo(responseInfo.requestOptions.core.mapToOfflineSdkType())
+            results = results
+                .map { OfflineSearchResult(it.rawSearchResult) }
+                .filter { it.name.isNotBlank() },
```

Comment:
Is it expected that results can have empty id? If it's a bug, it's better not to hide.

Reply:
Yes, SSDK-631 have examples of empty names. Not sure about `id` though, it's not empty.

Reply:
Where to fix it or hide is a separate conversation that we had with @mapbox-danny yesterday. Currently, the decision is to hide it until a proper fix is developed.

Reply:
Added assert